qiime/core/testing/type.py

```diff
@@ -0,0 +1,15 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import qiime.plugin
+
+
+IntSequence1 = qiime.plugin.SemanticType('IntSequence1')
+IntSequence2 = qiime.plugin.SemanticType('IntSequence2')
+Mapping = qiime.plugin.SemanticType('Mapping')
+FourInts = qiime.plugin.SemanticType('FourInts')
```

Comment:
Is it worth having composite types here? It would be relevant for the sake of integration testing, but we could also just rely on the unit-tests of qiime.core.type